rollup.config.mjs

```diff
@@ -0,0 +1,26 @@
+import del from 'rollup-plugin-delete'
+import ts from '@wessberg/rollup-plugin-ts'
+
+const isDevelopmentBuild = process.env.BUILD === 'development'
+
+export default {
+  input: 'src/index.ts',
+  output: {
+    dir: 'lib',
+    format: 'cjs',
+    sourcemap: !!isDevelopmentBuild,
+    preserveModules: true,
+    preserveModulesRoot: 'src'
+  },
+  external: ['async_hooks', 'events', 'worker_threads'],
```

Comment:
Could you explain why this is needed?  
Do we always need to remember what is used by our tool?  

---

Seems it prints a warning into the console when you remove some externals :thinking:  
Should we try something like this: https://github.com/stevenbenisek/rollup-plugin-auto-external ?

Alternatively IMO it should fail the build if something missing was detected, so we don't build something that is not usable

_sry if I misunderstood this part of rollup, but I personally never used it before. But I heard that it is currently the best lib-bundler :+1: (if you are not relay on `tsc` directly :wink:)_

Reply:
The build remain usable, missing external modules is just a warning. 
The poolifier code is very lean, it's not necessary to add another rollup plugin to build that external list module, it can maintained manually.